src/derivatives/imaging.md

```diff
@@ -147,8 +148,9 @@ Template:
 ```Text
 <pipeline_name>/
     sub-<label>/
-        anat|func|dwi/
-            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>][_desc-<label>]_mask.nii.gz
+        [ses-<label>/]
+            anat|func|dwi/
+                <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>][_desc-<label>]_mask.nii.gz
```

Comment:
```suggestion
                <source_entities>[_space-<space>][_res-<label>][_label-<label>][_desc-<label>]_mask.nii.gz
```

I don't know if we plan to support gifti or cifti masks. If so, then this snippet will need filename templates for those as well.

Reply:
I'd be happy to make this change, but then it also needs to be reflected in the YAML files. Before I can do that, though, a question:

I'm noticing that the `dwi` datatype also allows the file extensions `.bvec` and `.bval`. Are those relevant for masks? Or should only `.json`, `.nii` and `.nii.gz` be allowed?

Reply:
I don'tthink `.bvec` and `.bval` would apply to masks. @mattcieslak WDYT?